apps/constellation/constellation.cpp

```diff
@@ -222,7 +216,7 @@ Constellation::Constellation(CertificatePtr certificate, Config config)
   , storage_(std::make_shared<StorageUnitClient>(internal_muddle_.AsEndpoint(), shard_cfgs_,
                                                  cfg_.log2_num_lanes))
   , lane_control_(internal_muddle_.AsEndpoint(), shard_cfgs_, cfg_.log2_num_lanes)
-  , dag_{GenerateDAG(cfg_.features.IsEnabled("synergetic"), "dag_db_", true, certificate)}
+  , dag_{GenerateDAG("dag_db_", true, certificate)}
```

Comment:
While I agree that this should PR should enable this by default. I would still like the option to disable it if it starts causing issues - at least in the short term

Reply:
Done.